src/export/CodeSystemExporter.ts

```diff
@@ -68,13 +68,17 @@ export class CodeSystemExporter {
   }
 
   export(): Package {
-    for (const cs of this.tank.getAllCodeSystems()) {
+    const codeSystems = this.tank.getAllCodeSystems();
+    for (const cs of codeSystems) {
       try {
         this.exportCodeSystem(cs);
       } catch (e) {
         logger.error(e.message, cs.sourceInfo);
       }
     }
+    if (codeSystems.length > 0) {
+      logger.info('Finished exporting FSH for FHIR code systems.');
```

Comment:
I think this message, and all the similar messages, could be marked as "FSH Export: ", similar to how you marked some messages as "FSH Import: ".